.github/workflows/release-please.yml

```diff
@@ -0,0 +1,14 @@
+ name: release-please
+ on:
+   push:
+     branches:
+       - master
+ jobs:
+   release-please:
+     runs-on: ubuntu-latest
+     steps:
+       - uses: GoogleCloudPlatform/release-please-action@v2
+         with:
+           token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Note: we are using `NODE_PKG_RELEASE_TOKEN` in other repositories. 

https://github.com/netlify/cli/blob/073217f558411e92f4dc29793308c616649ad41f/.github/workflows/release-please.yml#L12

Reply:
Probably makes sense to do so here, but I'll need a repo admin to add that secret 👀

Reply:
https://github.com/netlify/engineering/issues/259 would be the ideal solution though.